frontend/src/RulePage.tsx

```diff
@@ -178,10 +178,17 @@ export function RulePage(props: any) {
     languagesTabs = metadataJSON.all_languages.map(lang => <Tab label={lang} value={lang}/>);
     metadataJSONString = JSON.stringify(metadataJSON, null, 2);
 
-    const coverageMapper = (key: any, version: any) => {
-      return (
-        <li>{key}: {version}</li>
-      )
+    const coverageMapper = (key: any, range: any) => {
+      if (typeof range === "string") {
+        return (
+          <li >{key}: {range}</li>
+        );
+      } else {
+        return (
+          <li><span style={{textDecorationLine: 'line-through'}}>{key}: {range['since']}</span>
+              <span>(until {range['until']}) </span></li>
```

Comment:
the `until` is a bit small here... I still see `sonar-abap 3.7` and my brain intuitively ignores what's with strikethrough because... well, it's cut text so it's not necessary.

So my brain sees "Covered Since until sonar-abap 7.3..."

Is it possible to make this w/o strikethrough and something like

"Not covered in ABAP anymore. Was covered from version X to version Y."

![image](https://user-images.githubusercontent.com/38876598/135301094-72f7b0e0-5142-4ad4-b653-fffd59a73f8d.png)

```suggestion
              <span>(was covered until {range['until']}, not covered anymore) </span></li>
```

Reply:
done.